### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by mgianatagh**
Remove unused code

### CODE REVIEW

**Review by bobdonat — COMMENTED** [review]
There are still some references to the istio stuff removed.
For example, 
verrrazzanomodels_types.go has the following reference:
 //See  verrazzano-crd-generator/pkg/apis/networking.istio.io/v1alpha3/virtual_service.go

README.md has a reference to pkg/clientistio

I would suggest doing a global search of istio in your branch to find any references that should be removed.

**Review by gilbode — APPROVED** [review]

**Comment by mgianatagh** [comment]
> There are still some references to the istio stuff removed.
> For example,
> verrrazzanomodels_types.go has the following reference:
> //See verrazzano-crd-generator/pkg/apis/networking.istio.io/v1alpha3/virtual_service.go
> 
> README.md has a reference to pkg/clientistio
> 
> I would suggest doing a global search of istio in your branch to find any references that should be removed.

Good catch, I had searched for file names with istio, but not the contents.  I think I got them all now.  I pushed the changes.

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,6 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.5.1
 	gopkg.in/yaml.v2 v2.2.8
-	istio.io/api v0.0.0-20200107183329-ed4b507c54e1
```
Comment (bobdonat) [inline go.mod:8]: Does this need to be removed from go.sum too?  Maybe you can delete go.sum and regenerate.

Comment (mgianatagh) [reply go.mod:8]: I updated the go.mod target to also do "go mod tidy".  That removed it from the go.sum.

**Review by bobdonat — APPROVED** [review]

**Pull request merged**